ui/lib/core/addon/components/form-field.js

```diff
@@ -19,13 +19,18 @@ import layout from '../templates/components/form-field';
  * @param [onChange=null] {Func} - Called whenever a value on the model changes via the component.
  * @param attr=null {Object} - This is usually derived from ember model `attributes` lookup, and all members of `attr.options` are optional.
  * @param model=null {DS.Model} - The Ember Data model that `attr` is defined on
+ * @param [disabled=false] {Boolean} - whether the field is disabled
+ * @param [showHelpText=true] {Boolean} - whether to show the tooltip with help text from OpenAPI
```

Comment:
these attributes should also be added to the `form-field.md` for Storybook :)

Reply:
Yep! I didn't regenerate / move things over yet because it's just for boolean checkboxes - I was going to add this after we pipe these items through to the rest of form-field (the TODO from the PR description).